<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>845</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1145</width>
    <height>845</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1145</width>
    <height>845</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt iRadarSimPro</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>12</y>
      <width>1143</width>
      <height>793</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Radar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="_RadarListView" name="radarListView">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>360</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>360</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Platform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="_PlatformTreeView" name="platformTreeView">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>360</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>360</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>720</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Trace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="_SituationGraphicView" name="situationGraphicsView">
         <property name="minimumSize">
          <size>
           <width>762</width>
           <height>762</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>762</width>
           <height>762</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addRadarAction"/>
   <addaction name="addTraceAction"/>
   <addaction name="addPlatformAction"/>
   <addaction name="clearAction"/>
   <addaction name="runAction"/>
   <addaction name="stopAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addRadarAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/images/addRadar.bmp</normaloff>:/img/images/addRadar.bmp</iconset>
   </property>
   <property name="text">
    <string>Add Radar</string>
   </property>
   <property name="toolTip">
    <string>Add Radar</string>
   </property>
  </action>
  <action name="addTraceAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/images/addTrace.BMP</normaloff>:/img/images/addTrace.BMP</iconset>
   </property>
   <property name="text">
    <string> Add Trace</string>
   </property>
   <property name="toolTip">
    <string> Add Trace</string>
   </property>
  </action>
  <action name="addPlatformAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/images/addPlatform.BMP</normaloff>:/img/images/addPlatform.BMP</iconset>
   </property>
   <property name="text">
    <string>Add Platform</string>
   </property>
   <property name="toolTip">
    <string>Add Platform</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/images/clear.bmp</normaloff>:/img/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>Clear Situation</string>
   </property>
   <property name="toolTip">
    <string>Clear Situation</string>
   </property>
  </action>
  <action name="runAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/images/run.bmp</normaloff>:/img/images/run.bmp</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/images/stop.bmp</normaloff>:/img/images/stop.bmp</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>_RadarListView</class>
   <extends>QListView</extends>
   <header>radarlistview.h</header>
  </customwidget>
  <customwidget>
   <class>_PlatformTreeView</class>
   <extends>QTreeView</extends>
   <header>platformtreeview.h</header>
  </customwidget>
  <customwidget>
   <class>_SituationGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>situationgraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
